_projects/2025/radis/Fast_Parsing.md

```diff
@@ -0,0 +1,73 @@
+---
+name: Fast parsing of large databases and execution bottlenecks
+desc: The conversion of large files from a compressed format to hdf5 should be accelerated.
+
+
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Knowledge of Pandas or other DataFrame libraries
+difficulty: medium
+issues:
+- https://github.com/radis/radis/issues/510
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - minouHub
+ - erwanp
+ - dcmvdbekerom
+initiatives:
+ - GSOC
+project_size:
+ - 175 h
+tags:
+# Different technologies needed
+ - Python
+ - Vaex
+ - Pandas
+ - Git
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - radis
+---
+
+
+#### Description
+
+The RADIS code was developed for the characterization of plasmas, flames and atmospheres. High-temperature spectral calculations require to resolve the shape of tens of millions of lines, which is the usual performance bottleneck. RADIS implements a new algorithm to compute these lineshapes, and is already one of the fastest line-by-line spectral codes available. It can also compute many different types of spectra (absorption / emission / equilibrium / nonequilibrium). In a typical calculation, a database of coefficients is loaded and these coefficients are multiplied according to physics laws to generate a set of linestrength in a spectrum. RADIS can handle different molecular databases such as HITRAN, HITEMP, EXOMOL, and GEISA. 
```

Comment:
```suggestion
The RADIS code was developed for the characterization of plasmas, flames and atmospheres.
High-temperature spectral calculations require to resolve the shape of tens of millions of lines, which is the usual performance bottleneck.
RADIS implements a new algorithm to compute these lineshapes, and is already one of the fastest line-by-line spectral codes available.
It can also compute many different types of spectra (absorption / emission / equilibrium / nonequilibrium).
In a typical calculation, a database of coefficients is loaded and these coefficients are multiplied according to physics laws to generate a set of line strengths in a spectrum.
RADIS can also handle different molecular databases such as HITRAN, HITEMP, EXOMOL, and GEISA.
```